### ISSUE TITLE
Private class fields don't work in a TypeScript file

### ISSUE THREAD

**Issue by fabis94** [unjs/jiti#23]
Am I correct in guessing that this lib doesn't load the private class properties proposal for TypeScript files? This file makes me think so: https://github.com/nuxt-contrib/jiti/blob/5d5ee2895d6670044bda368449557a6a131f7439/src/babel.ts#L19

Because of this (or another) issue I'm unable to use private class properties in my Nuxt.js app.

**Comment by fabis94** [comment]
The weird thing is that I'm using the latest version of @nuxt/babel-preset-app which supposedly does include this babel plugin

**Comment by pi0** [comment]
Hi @fabis94. Jiti uses an isolate build of babel to transform syntax. And the reason is excluded is that `@babel/plugin-transform-typescript` is supposed to work and locally testing this works (in a typescript file):

```ts
class Test {
 foo: string = 'bar'
}

console.log(new Test().foo)
```

Would you please share syntax used in `smartRuntimeConfig.ts` or if possible a reprodution?

**Comment by fabis94** [comment]
@pi0 Your example has public properties, `smartRuntimeConfig.ts` has private ones, like so:
```typescript
class SmartRuntimeConfig {
  #foo: string;

  constructor(foo: string) {
    this.#foo= foo;
  }
}
```

**Comment by pi0** [comment]
You are right @fabis94. Fixed in 1.6.4. You have to recreate lock file to get update.

**Comment by fabis94** [comment]
Thank you
